packages/lesswrong/server/rss.ts

```diff
@@ -16,8 +16,9 @@ import { fetchFragment } from './fetchFragment';
 import { addStaticRoute } from "./vulcan-lib/staticRoutes";
 import { createAnonymousContext } from "./vulcan-lib/query";
 
-Posts.addView('rss', Posts.views.new); // default to 'new' view for RSS feed
-Comments.addView('rss', Comments.views.recentComments); // default to 'recentComments' view for comments RSS feed
+// TODO: figure out what was going on here!
+// Posts.addView('rss', Posts.views.new); // default to 'new' view for RSS feed
+// Comments.addView('rss', Comments.views.recentComments); // default to 'recentComments' view for comments RSS feed
```

Comment:
I think the practical upshot of this is that there are views named "rss" which are aliases for the "new" view on posts, or the "recentComments" view on comments. Since RSS endpoints do use `viewTermsToQuery`, dropping these view-names would turn off RSS feeds for people subscribed to URLs containing those view-names. And "rss" seems to be the default if no view is specified.

Reply:
I see that "rss" was moved into posts/views.ts and comments/views.ts, which is probably where they should've been all along.

Reply:
If the import order was wrong this could theoretically be adding null as a view, but I checked `server.ts` and I believe the import order was such that it worked out okay. So we can probably just delete this TODO.

Reply:
Sorry, to clarify, the issue is that this might have been overwriting the view function assigned to `rss` in `posts/views` with `new` (whereas it was previously just a re-assignment of `community-rss`).